README.md

```diff
@@ -1 +1,411 @@
+
+#### Setup Azure ProviderConfig
+
+Before creating any resources, we need to create and configure an Azure cloud provider resource in Crossplane, which stores the cloud account information in it. All the requests from Crossplane to Azure Cloud will use the credentials attached to this provider resource. The following command assumes that you have a crossplane-azure-provider-key.json file that belongs to the account you’d like Crossplane to use.
+
+Now we’ll create our Secret that contains the credential and ProviderConfig resource that refers to that secret:
+
+```console
+kubectl create secret generic azure-account-creds -n upbound-system --from-file=credentials=./crossplane-azure-provider-key.json
+```
+
+```console
+cat > provider.yaml <<EOF
+---
+apiVersion: azure.crossplane.io/v1beta1
+kind: ProviderConfig
+metadata:
+  name: default
+  namespace: upbound-system
+spec:
+  credentials:
+    source: Secret
+    secretRef:
+      namespace: upbound-system
+      name: azure-account-creds
+      key: credentials
+EOF
+```
+
+Apply it to the cluster:
+```console
+kubectl apply -f "provider.yaml"
+```
+The output will look like the following:
+
+```shell
+provider.azure.crossplane.io/default created
+```
+
+Crossplane resources use the ProviderConfig named ```default``` if no specific ProviderConfig is specified, so this ProviderConfig will be the default for all Azure resources.
+
+### We are now ready to provision resources:
+
+#### Create Network Fabric
+
+The example network composition includes the creation of a Resource Group, Virtual Network, and Subnets:
+
+```console
+kubectl apply -f examples/network.yaml
+```
+
+Verify status:
+
+```console
+kubectl get claim
+kubectl get composite
+kubectl get managed
+```
+
+#### Invite App Teams to you Organization in Upbound Cloud
+
+1. Create a Team `team1`.
+1. Invite app team members and grant access to `Control Planes` and `Repositories`.
+
+App Dev/Ops: Consume the infrastructure you need using kubectl
+
+#### Join your Organization in Upbound Cloud
+
+1. **Join** your [Upbound Cloud](https://cloud.upbound.io/register)
+   `Organization`
+1. Verify access to your team `Control Planes` and Registries
+
+#### Provision a CompositePostgreSQLInstance in your team Control Plane GUI console
+
+1. Browse the available self-service APIs (XRDs) Control Plane
+1. Provision a `CompositePostgreSQLInstance` using the custom generated GUI for your
+Platform `Configuration`
+1. View status / details in your `Control Plane` GUI console
+
+#### Connect kubectl to your team Control Plane
+
+1. Connect `kubectl` to a `Control Plane` from the self-service GUI console.
+
+#### Provision a PostgreSQLInstance using kubectl
+
+```console
+kubectl apply -f examples/postgres-claim.yaml
+```
+
+Verify status:
+
+```console
+kubectl get claim
+kubectl get composite
+kubectl get managed
+```
+
+### Cleanup & Uninstall
+
+#### Cleanup Resources
+
+Delete resources created through the `Control Plane` Configurations menu:
```

Comment:
are these GUI deletion instructions correct? we can perhaps consider streamlining this README by only having kubectl instructions, instead of a mix of both kubectl and GUI.

Reply:
I did not test the GUI cleanup instructions. It sounds reasonable to me to just use the kubectl method in the guide.